CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Deprecated/Removed
 - The TO API `servers/totals` endpoint is now deprecated
+- Added a boolean to delivery service in Traffic Portal and Traffic Ops to enable EDNS0 client subnet at the delivery service level and include it in the cr-config.
```

Comment:
these should go under the `Added` section

Reply:
done. sorry about that